controllers/apartments_controllers.rb

```diff
@@ -0,0 +1,30 @@
+
+get '/apartments' do
+  @apartments =  Apartment.all
+  erb :index
+end
+
+get '/apartments/:id' do
+@id = params[:name]
+@apartment = Apartment.find(@id)
```

Comment:
You don't really need to save the ID to a variable anyway... Just do `@apartment = Apartment.find(params[:id])`